lib/elixir/lib/string.ex

```diff
@@ -505,13 +505,13 @@ defmodule String do
   ## Examples
 
       iex> String.normalize("yêṩ", :nfd)
-      "yêṩ"
+      "yêṩ"
 
       iex> String.normalize("leña", :nfc)
       "leña"
 
   """
-  @spec normalize(t, atom) :: boolean
+  @spec normalize(t, atom) :: t
```

Comment:
:information_desk_person: Don't know if this is the right place to change this but it seems incorrect.